relays/substrate-client/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "relay-substrate-client"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "1.3.4" }
+headers-relay = { path = "../headers-relay" }
+jsonrpsee = { git = "https://github.com/svyatonik/jsonrpsee.git", branch = "shared-client-in-rpc-api", default-features = false, features = ["http"] }
+log = "0.4.11"
+num-traits = "0.2"
+relay-utils = { path = "../utils" }
+
+# Supported Chains
+
+millau-runtime = { path = "../../bin/millau-runtime", optional = true }
+rialto-runtime = { path = "../../bin/rialto-runtime", optional = true }
+
+# Substrate Dependencies
+
+frame-system = { version = "2.0.0-rc6", tag = 'v2.0.0-rc6', git = "https://github.com/paritytech/substrate.git" }
+pallet-transaction-payment = { version = "2.0.0-rc6", tag = 'v2.0.0-rc6', git = "https://github.com/paritytech/substrate.git", optional = true }
+sp-core = { version = "2.0.0-rc6", tag = 'v2.0.0-rc6', git = "https://github.com/paritytech/substrate.git" }
+sp-keyring = { version = "2.0.0-rc6", tag = 'v2.0.0-rc6', git = "https://github.com/paritytech/substrate.git", optional = true }
+sp-runtime = { version = "2.0.0-rc6", tag = 'v2.0.0-rc6', git = "https://github.com/paritytech/substrate.git" }
```

Comment:
Until we have an automated check & formatting tool for this, I'm reluctant to address that. We have `cargo fmt` to avoid biases towards formatting in `.rs` files, and I really think we should do that for `.toml` as well, otherwise it's pretty futile.
Added an issue for this: https://github.com/paritytech/parity-bridges-common/issues/378